rmc-docs/src/tool-comparison.md

```diff
@@ -0,0 +1,20 @@
+# Comparison with other tools
```

Comment:
You could talk about the progression of: concrete inputs with testing, probabilistic inputs with fuzzing and symbolic inputs with model checking. This section has a few terms that need explaining: SMT, NP-complete.